### PR TITLE
MBL-727: Create splash screen and launch from Discovery Activity

### PR DESCRIPTION
**Pull request by leighdouglas**
# 📲 What

Added the android splash screen api to our app. 

# 🛠 How

Add the api, create the theme/style, add the manifest and call from onCreate before `super.onCreate(savedInstanceState)`
Splash screen is automatically dismissed once the first view element in discovery activity is drawn

# 👀 See

238782188-4b938c34-06bc-47c4-90cf-3e9be0cf628e

# 📋 QA

Make sure splash screen loads on devices both below and above api 33

# Story 📖

[MBL-727: Create splash screen and launch from Discovery Activity](https://kickstarter.atlassian.net/browse/MBL-727)

### CODE REVIEW

**Review by mtgriego — APPROVED** [review]
Tested and looking good!

**Comment by mtgriego** [comment]
You do seem to have a failing test though

File: app/src/main/java/com/kickstarter/ui/activities/DiscoveryActivity.kt [hunk 1]
```diff
@@ -5,8 +5,11 @@ import android.content.DialogInterface
 import android.content.Intent
 import android.graphics.drawable.Animatable
 import android.os.Bundle
+import android.view.View
+import android.view.ViewTreeObserver
```
Comment (ycheng-kickstarter) [inline app/src/main/java/com/kickstarter/ui/activities/DiscoveryActivity.kt:9]: Were these added imports all necessary? I don't see them used.

**Review by Arkariang — APPROVED** [review]
lgtm! 💖

**Review by ycheng-kickstarter — APPROVED** [review]

**Pull request merged**